src/resources/lib/abstract-resource.js

```diff
@@ -251,8 +253,13 @@ module.exports = class AbstractResource extends Restypie.Resources.AbstractCoreR
     if (this.defaultSelect) {
       this.defaultSelect.forEach(keyItem => {
         if (!schema.hasOwnProperty(keyItem)) {
-          throw new Error('Schema doesnt have this property ' + keyItem);
+          throw new Error(`Schema doesnt have this property ${keyItem}`);
         }
+        FIELD_AUTH_METHODS.forEach(permCheck => {
+          if (schema[keyItem].hasOwnProperty(permCheck)) {
```

Comment:
This should only check for canRead, `defaultSelect` has nothing to do with writing